chapter_recurrent-neural-networks/lang-model.md

```diff
@@ -118,7 +118,7 @@ As we can see, the most popular words are actually quite boring to look at. In t
 ```{.python .input}
 wordcounts = [count for _,count in counter.most_common()]
 d2l.use_svg_display()
-d2l.plt.loglog(wordcounts);
```

Comment:
; is used to prevent display the return of loglog

Reply:
Thanks. It is fixed now.